bin/hoc-2015-nov-9-mailing/send-general-final-real-mails

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# ubuntu@production-console:~/production/bin/hoc-2015-nov-9-mailing$ wc -l general-deduped-remainder.csv
+# 1261412 general-deduped-remainder.csv
+# 1,261,412
+
+./send-to-mailing-list --start       0 --limit 100000 11-9-hoc-general hoc-2015-nov-9-mailing/general-deduped-test-remainder.csv
```

Comment:
ok so you need to fix this line and the rest? (this line says `hoc-2015-nov-9-mailing/general-deduped-test-remainder.csv` and it looks like you want `hoc-2015-nov-9-mailing/general-deduped-remainder.csv` ?)